rotkehlchen/api/rest.py

```diff
@@ -4317,3 +4317,12 @@ def get_evm_counterparties_details(self) -> Response:
             )),
             status_code=HTTPStatus.OK,
         )
+
+    def get_history_event_locations(self) -> Response:
```

Comment:
Remove this. This function makes no sense. Just do a process_result() location directly in resource and don't use the misleading "events" related name